packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -79,12 +89,20 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
       theme: theme!
     });
     const divButtonProps = buttonProps ? getNativeProps<React.HTMLAttributes<HTMLDivElement>>(buttonProps, divProperties) : undefined;
-
+    const thumbButton = (
+      <span
+        ref={this._thumb}
+        className={classNames.thumb}
+        style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)}
+        id={this._buttonId}
+        aria-labelledby={showThumbTooltip ? this._hostId : undefined}
+      />
+    );
     return (
       <div className={classNames.root}>
-        {label && (
+        {showValue && label && (
```

Comment:
This is changing behavior expected form previous implementation. Before `label` was rendered independently in regards if user provided `showValue` or not. He could render a label without the value and a value without the label. Now it seems if user needs a label he is forced to show the value too and you still did not used `valueLabel` className anywhere 🤔

Reply:
oh I see what you mean I will change that right now